griptape/tools/griptape_cloud_knowledge_base_client/tool.py

```diff
@@ -52,4 +52,6 @@ def _get_knowledge_base_description(self) -> str:
             if "description" in response:
                 return response["description"]
             else:
-                raise ValueError(f'Error getting Knowledge Base description: {response["message"]}')
+                raise ValueError(
+                    "No description found for Knowledge Base. Please set a description, or manually set the `GriptapeCloudKnowledgeBaseClient.description` attribute."
```

Comment:
maybe throw the id in the message?

Reply:
Does that come back in the response?

Reply:
not sure, but cant you pull it from `self. knowledge_base_id`?

Reply:
Oh duh